crates/hyperswitch_connectors/src/connectors/fiuu/transformers.rs

```diff
@@ -1921,7 +1921,7 @@ pub enum FiuuWebhooksRefundType {
 }
 
 #[derive(Debug, Deserialize, Serialize, Clone)]
-pub struct FiuuWebhookSignauture {
+pub struct FiuuWebhookSignature {
```

Comment:
If you changed this type name but you didn't have to change the usage anywhere else, is this type even being used?

Reply:
by the looks of it, this struct is unused (i did all this change after verifying):
<img width="724" alt="image" src="https://github.com/user-attachments/assets/91914a24-ca04-48b2-b1bf-8bd9e33a29cf" />

Reply:
checked with @awasthi21, this struct itself is redundant.